doc/user/content/sql/system-catalog/mz_catalog.md

```diff
@@ -479,9 +479,20 @@ Field            | Type                 | Meaning
 
 ### `mz_storage_usage`
 
-The `mz_storage_usage` table describes the storage utilization of each table,
-source, and materialized view in the system. Storage utilization is assessed
-approximately every hour.
+{{< if-released "v0.111" >}}
```

Comment:
Why does this need to be gated by the release? It's describing a true thing from before 0.111 too right?

Reply:
Ah yeah I refactored this doc. The `if-released` bit should apply to the paragraph that references the new view (`mz_internal.mz_recent_storage_usage`). Thanks for catching that.